soc/arm/silabs_sim3/sim3u/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) 2023 GARDENA GmbH
+#
+# SPDX-License-Identifier: Apache-2.0
+
+zephyr_sources(soc.c)
+
+set(CROSSBAR_CONFIG_H ${CMAKE_BINARY_DIR}/zephyr/include/generated/silabs_crossbar_config.h)
```

Comment:
this is being refactored now to include `zephyr/` prefix https://github.com/zephyrproject-rtos/zephyr/pull/63973